Modules/Chatroom/classes/class.ilChatroom.php

```diff
@@ -809,6 +816,7 @@ public function isOwnerOfPrivateRoom(int $user_id, int $proom_id) : bool
         return false;
     }
 
+    // CR: Type of parameter $sender is missing and the $DIC can be rmoved, because $DIC is not used.  
```

Comment:
I removed the `$DIC`. The type cannot be explicitly specified. This will be possible with PHP 8 (`Union Types`), but we'll have to support PHP 7.4. I used `PHPDoc` to describe the types.